services/notifications/src/simcore_service_notifications/services/rabbitmq.py

```diff
@@ -0,0 +1,30 @@
+from collections.abc import AsyncIterator
+from typing import cast
+
+from fastapi import FastAPI
+from fastapi_lifespan_manager import State
+from servicelib.rabbitmq import RabbitMQRPCClient, wait_till_rabbitmq_responsive
+from settings_library.rabbit import RabbitSettings
+
+from ..core.settings import ApplicationSettings
+
+
+async def rabbitmq_lifespan(app: FastAPI) -> AsyncIterator[State]:
+    settings: ApplicationSettings = app.state.settings
+    rabbit_settings: RabbitSettings = settings.NOTIFICATIONS_RABBITMQ
+    app.state.rabbitmq_rpc_server = None
+
+    await wait_till_rabbitmq_responsive(rabbit_settings.dsn)
+
+    app.state.rabbitmq_rpc_server = await RabbitMQRPCClient.create(
```

Comment:
why do we need an RPC server here? 
The idea was to create a pub-sub message system, not rpc connections.

Reply:
I do not understand this statement. I think we need a RPC interface in order to interact with the notifications services? I thought we had RPC and HTTP interfaces.

What is pub-sub? Ut is just a pattern, we have no such type of interface currently. 

Also to me pub-sub has an issue, since it means that the sub part is optional which does not create any guarantee for message delivery.